File: library/src/main/java/com/example/library/dto/response/LoanSlipResponse.java
package com.example.library.dto.response;

import java.time.LocalDate;
import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class LoanSlipResponse {
	
    private Integer id;

    private String receiptNumber;
    
    private Integer createdByAccountId;
    
    private String createdByUsername;

    private Integer bookId;
    
    private String bookName;
    
    private String bookCode;

    private Integer memberId;
    
    private String memberName;
    
    private String memberCardNumber;

    private Integer state;

    private LocalDate borrowDate;
    
    private LocalDate dueDate;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;  
}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/LoanSlipResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDate
import java.time.LocalDateTime


data class LoanSlipResponse(
     val id: Int,
     val receiptNumber: String,
     val createdByAccountId: Int,
     val createdByUsername: String,
     val bookId: Int,
     val bookName: String,
     val bookCode: String,
     val memberId: Int,
     val memberName: String,
     val memberCardNumber: String,
     val state: Int,
     val borrowDate: LocalDate,
     val dueDate: LocalDate,
     val updatedAt: LocalDateTime,
     val deleteAt: LocalDateTime?
)